common/src/config/mod.rs

```diff
@@ -0,0 +1,20 @@
+use lazy_static::lazy_static;
+use std::sync::RwLock;
+use config::{Config, File};
+
+lazy_static! {
+	pub static ref CONFIG: RwLock<Config> = {
```

Comment:
Why do we need an `RWLock`?
The config is loaded here and then all access is done reading. So it could just be an immutable config, I guess.

Reply:
This is a global variable for the whole project. If we have different binaries, e.g. webservices, CLI, administration, we may want to load additional config from other paths and sources. We also may want to dynamically update the config during runtime. For this we need an RWLock.